Formula/open-mpi.rb

```diff
@@ -1,8 +1,26 @@
 class OpenMpi < Formula
   desc "High performance message passing library"
   homepage "https://www.open-mpi.org/"
-  url "https://www.open-mpi.org/software/ompi/v3.1/downloads/openmpi-3.1.0.tar.bz2"
-  sha256 "b25c044124cc859c0b4e6e825574f9439a51683af1950f6acda1951f5ccdf06c"
+
+  stable do
```

Comment:
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Do `patch do` blocks outside of any `SoftwareSpec` block apply the patches they pull in to the `stable` spec?  I couldn't remember if that was the case off of the top of my head…